HTML/Nonogram translated (Čeština — Malované Křížovky) optimalizováno.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<title>Malovaná Křížovka — Keep Talking and Nobody Explodes Mod</title>
```

Comment:
please take the file name title from here instead

Reply:
also changing pdf names, thx for reminder